### PR TITLE
Wip/andrej.picej@norik.com/bspimx8 m 3351

### PR DESCRIPTION
**Pull request by andrejpicej**
Need to exclude PEB-AV-12 and tripple display support from next NXP BSP release.

Hope this makes sense. Not sure if this is easier then copying the whole display chapter to the base file of the release and modifying it there.

### CODE REVIEW

File: source/bsp/imx8/imx8mp/display.rsti [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Display
 -------
 
+.. supported-displays-marker-start
```
Comment (BHahn42618) [inline source/bsp/imx8/imx8mp/display.rsti:4]: This is more about the supported display interfaces than the supported displays themselves.
```suggestion
.. supported-display-interfaces-marker-start

File: source/bsp/imx8/imx8mp/display.rsti [hunk 2]
```diff
@@ -15,6 +17,8 @@ LVDS1     |sbc|                    disabled if PEB-AV-10 overlay is used
 MIPI      PEB-AV-12 (MIPI to LVDS) imx8mp-phyboard-pollux-peb-av-012.dtbo
 ========= ======================== ======================================
 
+.. supported-displays-marker-end
```
Comment (BHahn42618) [inline source/bsp/imx8/imx8mp/display.rsti:20]:
```suggestion
.. supported-display-interfaces-marker-end

File: source/bsp/imx8/imx8mp/display.rsti [hunk 3]
```diff
@@ -24,7 +28,7 @@ HDMI is always enabled in the devicetree. The other interfaces can be enabled
 with Device Tree Overlay.
 
 The default-enabled Interfaces are HDMI and LVDS0 (PEB-AV-010). We support a
-10'' edt,etml1010g0dka display for the PEB-AV-10 and PEB-AV-12.
+10'' edt,etml1010g0dka display for the |10inch-displays|.
```
Comment (BHahn42618) [inline source/bsp/imx8/imx8mp/display.rsti:31]: This is not about the 10inch-displays we support. In the future we could support other displays. This is about the lvds display adapters we support.
```suggestion
10'' edt,etml1010g0dka display for the |lvds-display-adapters|.

File: source/bsp/imx8/imx8mp/head.rst [hunk 4]
```diff
@@ -112,6 +112,7 @@
 .. |ref-jp4| replace:: :ref:`JP4 <imx8mp-head-components>`
 .. |ubootexternalenv| replace:: U-boot External Environment subsection of the
    :ref:`device tree overlay section <imx8mp-head-ubootexternalenv>`
+.. |10inch-displays| replace:: PEB-AV-10
```
Comment (BHahn42618) [inline source/bsp/imx8/imx8mp/head.rst:115]:
```suggestion
.. |lvds-display-adapters| replace:: PEB-AV-10

File: source/bsp/imx8/imx8mp/head.rst [hunk 5]
```diff
@@ -514,6 +515,24 @@ Device Tree Audio configuration:
 .. include:: /bsp/peripherals/video.rsti
 
 .. include:: display.rsti
+   :end-before: .. supported-displays-marker-start
```
Comment (BHahn42618) [inline source/bsp/imx8/imx8mp/head.rst:517]:
```suggestion
   :end-before: .. supported-display-interfaces-marker-start

File: source/bsp/imx8/imx8mp/head.rst [hunk 6]
```diff
@@ -514,6 +515,24 @@ Device Tree Audio configuration:
...
+Interface Expansion                devicetree overlay
+========= ======================== ======================================
+HDMI      |sbc|                    no overlay needed (enabled by default)
+LVDS0     PEB-AV-10                |dtbo-peb-av-10|
+                                   (loaded by default)
+LVDS1     |sbc|                    disabled if PEB-AV-10 overlay is used
+========= ======================== ======================================
+
+.. include:: display.rsti
+   :start-after: .. supported-displays-marker-end
```
Comment (BHahn42618) [inline source/bsp/imx8/imx8mp/head.rst:533]:
```suggestion
   :start-after: .. supported-display-interfaces-marker-end

File: source/bsp/imx8/imx8mp/pd23.1.0.rst [hunk 7]
```diff
@@ -104,6 +104,7 @@
    Starting with BSP-Yocto-i.MX8MP-PD22.1.1 we have to switch from PWM fan
    to GPIO fan due to availability. The PWM fan will not be supported
    anymore and will not function with the new release.
+.. |10inch-displays| replace:: PEB-AV-10 and PEB-AV-012
```
Comment (BHahn42618) [inline source/bsp/imx8/imx8mp/pd23.1.0.rst:107]:
```suggestion
.. |lvds-display-adapters| replace:: PEB-AV-10 and PEB-AV-012

**Review by BHahn42618 — CHANGES_REQUESTED** [review]

**Review by tremmet — APPROVED** [review]

**Pull request merged**